ios/test_app.rb

```diff
@@ -429,6 +429,7 @@ def use_test_app_internal!(target_platform, options)
   end
 
   post_install do |installer|
+    options[:post_install]&.call(installer) if options[:post_install]
     react_native_post_install&.call(installer)
```

Comment:
Can we reorder this so that the React Native one always runs first? I'm not sure if people rely on it being run first to modify the environment etc.
```suggestion
    react_native_post_install&.call(installer)
    options[:post_install]&.call(installer) if options[:post_install]
```

Reply:
@tido64 thanks - applied